debian/changelog

```diff
@@ -1,11 +1,7 @@
-kodi-screensavers-rsxs (1.0.1-1~raring) raring; urgency=low
+kodi-screensavers-rsxs (3.0.0-0~cosmic) cosmic; urgency=low
```

Comment:
This file shouldn't be committed, because it is autogenerated.

Reply:
Was wrongly added to commit, becomes removed.